lib/node/localNode.go

```diff
@@ -30,6 +30,35 @@ type LocalNode struct {
 	validators map[ /* Node.Address() */ string]*Validator
 }
 
+func NewLocalNode(address string, endpoint *sebakcommon.Endpoint, alias string) (n *LocalNode, err error) {
+	if len(alias) < 1 {
+		alias = MakeAlias(address)
+	}
+
+	if _, err = keypair.Parse(address); err != nil {
+		return
+	}
+
+	n = &LocalNode{
+		state:      NodeStateNONE,
+		alias:      alias,
+		address:    address,
+		endpoint:   endpoint,
+		validators: map[string]*Validator{},
+	}
+
+	return
+}
+
+func NewLocalNodeFromString(b []byte) (*LocalNode, error) {
```

Comment:
This func have other functionality?? 
it seems just deserialize object isn't it?

Reply:
Oh yeah, it's removed in a later commit actually (it's not added, just moved)